test/e2e/scheduling/predicates.go

```diff
@@ -515,11 +518,14 @@ var _ = SIGDescribe("SchedulerPredicates [Serial]", func() {
 		gomega.Expect(deployedPod.Spec.NodeName).To(gomega.Equal(nodeName))
 	})
 
-	// 1. Run a pod to get an available node, then delete the pod
-	// 2. Taint the node with a random taint
-	// 3. Try to relaunch the pod still no tolerations,
-	// and the pod's nodeName specified to the name of node found in step 1
-	ginkgo.It("validates that taints-tolerations is respected if not matching", func() {
+	/*
+		Release : v1.16
+		Testname : Scheduler, taints-tolerations not matching
+		Description : After removing the pod from the available node, verify that the
+		pod(without toleration against node taint) is not scheduled when the node is tainted with a
+		random taint. The pod MUST be scheduled onto the node when the taint has been removed.
+	*/
+	framework.ConformanceIt("validates that taints-tolerations is respected if not matching", func() {
```

Comment:
This test cannot be promoted because it uses `WaitForSchedulerAfterAction` which uses `scheduleFailureEvent` which checks `Event.Type` and `Event.Reason` fields

>Examples of tests which are not eligible to conformance:
- anything that checks specific Events are generated, as we make no guarantees about the contents of events, nor their delivery

Reply:
Thanks for the review. I have now removed the concerned test from promotion request int he recent commit.